catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -280,10 +289,29 @@ private String getTotalAmount(long hits) {
     private QueryRequest getIntendedQuery(FilterBuilder filterBuilder, boolean isRequestForTotal)
             throws InterruptedException {
 
-        Filter filter = filterBuilder.attribute(Metacard.ID).is().like().text(WILDCARD);
+        Filter filter = filterBuilder.allOf(filterBuilder.attribute(Metacard.ID)
```

Comment:
It looks like there's some duplication you could eliminate in these two filters.